markdown.config.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path');
```

Comment:
We should place this under `scripts/generate-markdown/markdown.config.js`. We've followed this pattern for repos with release scripts

Reply:
I moved the script and altered it slightly to work from the new location. Works on Mac and Windows. Let me know if that is what you had in mind!